.github/workflows/linux-eic-shell.yml

```diff
@@ -679,6 +700,26 @@ jobs:
         path: ref/
         name: rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root
         if_no_artifact_found: warn
+    - name: Compare to previous artifacts
+      uses: eic/run-cvmfs-osg-eic-shell@main
+      if: steps.download_previous_artifact.outputs.found_artifact == 'true'
+      with:
+        platform-release: "${{ env.platform-release }}"
+        setup: "/opt/detector/epic-${{ env.detector-version }}/setup.sh"
+        run: |
+          pip install 'pygithub>=2' 'bokeh>=3'
+          export PYTHONPATH=$HOME/.local/lib/python3.10/site-packages:$PYTHONPATH
+          mkdir capybara-reports
+          capybara bara ref/rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root
+          mv capybara-reports rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}
+          touch .rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}
```

Comment:
What is this for?

Reply:
"If multiple paths are provided as input, the least common ancestor of all the search paths will be used as the root directory of the artifact." (https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions)

If you upload an artifact for path `rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}/`, it just uploads the content of that directory. Since all file names in that directory are the same for different jobs, they end up overwriting each other. By adding a small hidden file one level up, we ensure that the directory is stored as well.